velox/exec/HashTable.cpp

```diff
@@ -1812,19 +1858,25 @@ int32_t HashTable<ignoreNullKeys>::listJoinResultsNoDuplicates(
     JoinResultIterator& iter,
     bool includeMisses,
     folly::Range<vector_size_t*> inputRows,
-    folly::Range<char**> hits) {
+    folly::Range<char**> hits,
+    uint64_t fixedSizeListColumnsSizeSum,
+    uint64_t maxBytes) {
   int32_t numOut = 0;
-  auto maxOut = inputRows.size();
+  const auto maxOut = std::min(
```

Comment:
Now this function only applies for the case: (1) table has no rows with duplicate join keys, (2) all list columns are fixed size? 
Shall we rename to
```
s/listJoinResultsNoDuplicates/listJoinResultsWithFixSizeColumnsAndNoDuplicateJoinKeys/
```

Reply:
Maybe just call it listJoinResultsFastPath and let comments do the explanation.